src/Routes.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import NotFound from './pages/notFound';
+import App from './components/app';
+import Privacy from './pages/privacy';
+
+const Routes = () => (
+  <Switch>
+    <Route exact path="/" component={App} />
+    <Route exact path="/privacy" component={Privacy} />
+    <Route component={NotFound} />
```

Comment:
hi @minega25 , i do think ist the best practice letting the user have a look of the URL he/she typed which is not found instead of redirecting to not found routes